src/main/java/com/tmh/entities/OrderItem.java

```diff
@@ -41,12 +41,25 @@
 	@Column(name = "quantity")
 	private int quantity;
 	
+	@Column(name = "deleted")
+	private Integer isDeleted;
+	
 	@Column(name = "created_at")
 	@CreationTimestamp
 	private LocalDateTime createDateTime;
 	
 	@Column(name = "updated_at")
 	@UpdateTimestamp
 	private LocalDateTime updateDateTime;
+	
+	public String getDeletedString() {
+		if (this.isDeleted == 0) {
+			return "NOT DELETED";
```

Comment:
Giá trị ni và delete lặp nhìu lần, chú khai báo 1 thèn enum dùng cho nó tiện nhé,
ko thì phải dùng constant đó

public enum StatusDeleteType {
	 Active(0), DELETED(1)

	int type;

	private StatusDeleteType(int type) {
		this.type = type;
	}

	public int getType() {
		return type;
	}

}